app/main.py

```diff
@@ -1,6 +1,14 @@
-from typing import Callable
+from typing import Callable, Any
 
 
-def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+def cache(func: callable) -> Callable:
```

Comment:
The type hint for `func` should be `Callable` with a capital 'C'. The correct import from `typing` is `Callable`, not `callable`.